_tests/browser_tests/page_objects/page_careers-students-and-graduates.js

```diff
@@ -0,0 +1,10 @@
+function careers_studentsAndGraduates() {
```

Comment:
The underscore in the function name will give a linter error. If we want to make underscores a convention in the test JS there just needs to be an eslintrc override file when https://github.com/cfpb/cfgov-refresh/issues/723 is closed.